tutorials/shading/shading_language.rst

```diff
@@ -947,13 +954,13 @@ Vertex Built-Ins
 +--------------------------------+----------------------------------------------------------------+
 | in bool **AT_LIGHT_PASS**      | True if this is a light pass (for multi-pass light rendering). |
 +--------------------------------+----------------------------------------------------------------+
-| inout vec2 **VERTEX**          | Vertex in image space.                                         |
+| out vec2 **VERTEX**            | Vertex in image space.                                         |
 +--------------------------------+----------------------------------------------------------------+
-| inout vec2 **UV**              | UV.                                                            |
+| out vec2 **UV**                | UV.                                                            |
 +--------------------------------+----------------------------------------------------------------+
-| inout vec4 **COLOR**           | Color from vertex primitive.                                   |
+| out vec4 **COLOR**             | Color from vertex primitive.                                   |
 +--------------------------------+----------------------------------------------------------------+
-| out vec2 **POINT_SIZE**        | Point size for point drawing.                                  |
+| out float **POINT_SIZE**       | Point size for point drawing.                                  |
```

Comment:
So POINT_SIZE was converted from `vec2` to `float`? Just asking for confirmation, I have no idea.

Reply:
Yeah

Reply:
According to shader_types.cpp it have TYPE_FLOAT, some types was also incorrect and I changed them